packages/labs/gen-utils/src/test/package-utils_test.ts

```diff
@@ -4,85 +4,93 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
```

Comment:
Changes here are just switching the test file to use FilesystemTestRig (missed that change in the PR that added it), and add tests for `packPackage`.